sdk/eventhub/eventhubs-checkpoint-blob/test/blob-partition-manager.spec.ts

```diff
@@ -0,0 +1,271 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import chai from "chai";
```

Comment:
Awesome test cases, this makes me happy :)

Reply:
Agreed!